.github/ISSUE_TEMPLATE/config.yml

```diff
@@ -0,0 +1,8 @@
+blank_issues_enabled: false
```

Comment:
Dunno if this is super desirable. It also doesn't really work currently, there are all sorts of holes in GH's implementation. I figure between bug reports, feature requests, and the Q&A section all major bases are covered for now.

Reply:
It is not working currently because we're not in the master branch

Reply:
Hm, not sure. It does work on Paper's repo for most of the workarounds I was aware of, they may have fixed them in the past few months. New accounts used to bypass it, as did certain direct links.